softwerkskammer/lib/dashboard/views/index.pug

```diff
@@ -13,13 +13,15 @@ block title
 block content
   .page-header
     h2 #{t('dashboard.dashboard')}
-    !=t('dashboard.description', { postProcess: 'pug' })
-    .row
-      .col-sm-6
-        +my_activities
-    +my_groups
+      small
+        p #{t('dashboard.description')}
```

Comment:
Lassen wir das "p" dann doch stehen? Ohne "p" steht die description eben in der selben Zeile (und mit wenig bis gar keinem Abstand, warum das auf anderen Seiten, z. B. im Adminbereich habe ich noch nicht ganz verstanden, nicht so ist. Das ist auf Agora nicht ganz einheitlich: auf einer Acitivity-Seite steht die graue Unterüberschrift unterhalb der großen (aber mit einem "br" in die nächste Zeile gezwungen).